v2.0/ping/README.md

```diff
@@ -0,0 +1,122 @@
+# Ping
+
+This is the simpest example for Singularity Compose to generate two containers
+with an alpine base, and then shell into one to ping the other.
+You'd first want to build the containers:
+
+```bash
+$ singularity-compose build
+```
+
+Note that since this example uses fakeroot for networking, you likely would need
+to update your configuration for your user as follows:
+
+```bash
+$ sudo singularity config fakeroot --add $USER
+```
+
+The above command would add _your_ username. You'll also need to update the fakeroot
+network configuration at `/usr/local/etc/singularity/network/40_fakeroot.conflist`
+(or replace with the prefix where you installed Singularity):
+
+```json
+{
+    "cniVersion": "0.4.0",
+    "name": "fakeroot",
+    "plugins": [
+        {
+            "type": "bridge",
+            "bridge": "sbr0",
+            "isGateway": true,
+            "ipMasq": true,
+            "ipam": {
+                "type": "host-local",
+                "subnet": "10.22.0.0/16",
+                "routes": [
+                    { "dst": "0.0.0.0/0" }
+                ]
+            }
+        },
+        {
+            "type": "firewall"
+        },
+        {
+            "type": "portmap",
+            "capabilities": {"portMappings": true},
+            "snat": true
+        }
+    ]
+}
+``` 
+
+See [here](https://sylabs.io/guides/3.7/user-guide/cli/singularity_config_fakeroot.html) for
+more details. Otherwise, you can remove the "start" sections and the client
+will ask you for sudo. Once this is done, bring them up (debug is used to show
+commands):
+
```

Comment:
I discovered one additional requirement in my environment that may be worth documenting. It may not be relevant for everyone so take it with a grain of salt.

My network has firewall rules on the router, which someone else configures, and another separate but matching set of rules on the host machine. In order for my containers to resolve hostnames, I needed to configure the firewall public zone to add IP masquerading with these commands:

```
sudo firewall-cmd --add-masquerade --zone=public --permanent
sudo firewall-cmd --reload
```
 See this issue for reference: https://github.com/hpcng/singularity/issues/5866

Reply:
This definitely isn't a problem for me, but I think more documentation (even for edge cases) is better than not having it - the small number of users that have this issue will be helped! I'll definitely add some comments to the PR.

Reply:
This is added!